pkg/gatherers/clusterconfig/sap_vsystem_iptables_logs.go

```diff
@@ -15,26 +15,34 @@ import (
 	"github.com/openshift/insights-operator/pkg/record"
 )
 
-// GatherSAPVsystemIptablesLogs collects logs from SAP vsystem-iptables containers
+// GatherSAPVsystemIptablesLogs Collects logs from SAP `vsystem-iptables` containers
 // including one from license management pods with the following substring:
 //   - "can't initialize iptables table",
 //
-// **Conditional data**: This data is collected only if the "installers.datahub.sap.com" resource is found in the cluster.
+// ### API Reference
+// - https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/pod_expansion.go#L48
+// - https://docs.openshift.com/container-platform/4.6/rest_api/workloads_apis/pod-core-v1.html#apiv1namespacesnamespacepodsnamelog
 //
-// The Kubernetes API:
+// ### Sample data
+// - docs/insights-archive-sample/config/pod/sdi/logs/license-manager-da1d2e8fadfb8dd7022f08-4hjh7-6887768c5b-qzxb6/errors.log
 //
-//	https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/pod_expansion.go#L48
+// ### Location in archive
+// | Version   | Path														|
+// | --------- | --------------------------------------------------------	|
+// | >= 4.8.2  | config/pod/{namespace}/logs/{pod-name}/errors.log 			|
 //
-// Response see:
+// ### Config ID
+// `clusterconfig/sap_license_management_logs`
 //
-//	https://docs.openshift.com/container-platform/4.6/rest_api/workloads_apis/pod-core-v1.html#apiv1namespacesnamespacepodsnamelog
+// ### Released version
+// - 4.8.2
 //
-// * Location in archive: config/pod/{namespace}/logs/{pod-name}/errors.log
-// * Id in config: clusterconfig/sap_license_management_logs
-// * Since versions:
-//   - 4.6.25+
-//   - 4.7.5+
-//   - 4.8+
+// ### Backported versions
+// - 4.7.5+
+// - 4.6.25+
+//
+// ### Notes
+// **Conditional data**: This data is collected only if the `installers.datahub.sap.com` resource is found in the cluster.
```

Comment:
Same thing here. It isn't true for all resources? Do we really need to have it here?

Reply:
I think it's an important information or not?

Reply:
Well... It is... I guess, but... why don't we have it in any other gatherer? Don't we have any resources that could exist only if the resource is installed in the cluster? 🤔

Reply:
Ahh now I see what you mean. This would mean to introduce a new type of condition (something like ResourceAvailable), right? Not sure it's worth investigating....we can create a spike for that.

Reply:
I don't think this is useful for rules as long as "no gatherer errors in gathers.json" and "no data in the archive" means "no resources on the cluster". Rules that need some resources will just get skipped if an archive wouldn't contain any.

Reply:
Yes, I think so.